source/overview/license-and-subscription.rst

```diff
@@ -0,0 +1,126 @@
+Licensing and Subscription
+----------------------
+
+Mattermost offers a secure self-service portal where you can manage your Mattermost Enterprise subscriptions for your on-premises deployments.
+All your licenses are listed in the portal, including their start and end dates, providing you with an at-a-glance overview of your account, making it easier to manage purchases and renewal dates.
```

Comment:
```suggestion
All your licenses are listed in the portal, including their start and end dates, providing you with an at-a-glance overview of your account, making it easier to manage purchases and renewal dates.
```
@rbradleyhaas, is this accurate for all licenses/customers?

Reply:
It is not accurate for Phase 1. Only licenses purchased through the portal will be available.